pkg/core/resources/apis/mesh/dataplane_validator_test.go

```diff
@@ -195,27 +195,6 @@ var _ = Describe("Dataplane", func() {
                 - field: networking
                   message: inbound cannot be defined both with gateway`,
 		}),
-		Entry("networking: invalid address", testCase{
```

Comment:
Hmm, are we saying all stings are valid addresses now?

Reply:
I found regex for domain names, I think I can check against it:
`^([a-zA-Z0-9_]{1}[a-zA-Z0-9_-]{0,62}){1}(\.[a-zA-Z0-9_]{1}[a-zA-Z0-9_-]{0,62})*[\._]?$`

Reply:
https://www.golangprograms.com/regular-expression-to-extract-dns-host-name-or-ip-address-from-string.html

Reply:
There is only IP address in the provided example

Reply:
I bet someone will try to put there `http://` or port. Please validate that it is a hostname or ip

Reply:
Fixed, validate using regex